Dockerfile

```diff
@@ -1,61 +1,55 @@
 FROM ubuntu:14.04
-MAINTAINER Gizra
+MAINTAINER Gizra <info@gizra.com>
 
-# Update and install packages
+# Disable interactive mode
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
ok, this seems to be used in [other](https://github.com/elgalu/docker-selenium/blob/master/Dockerfile#L35) places as-well